### PR TITLE
Feature/boat bay support

### PR DESCRIPTION
**Pull request by sophiejanssens**
Boat bay is still in development

### CODE REVIEW

File: boat-maven-plugin/src/main/java/com/backbase/oss/boat/BoatBayRadio.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+    boatbayUploadSpecClient = configureApiClient(this.clientBasePath).buildClient(UploadPluginApi.class);
+  }
+
+  private ApiClient configureApiClient(String clientBasePath){
+
+    if (readTimeout == null){
+      BasicAuthRequestInterceptor basicAuthRequestInterceptor = new BasicAuthRequestInterceptor("admin","admin");
+      ApiClient apiClient = new ApiClient().setBasePath(clientBasePath);
+//      apiClient.addAuthorization("Basic",basicAuthRequestInterceptor);
```
Comment (christopherm-backbase) [inline boat-maven-plugin/src/main/java/com/backbase/oss/boat/BoatBayRadio.java:59]: How is auth working now?

Comment (dhananjay12) [reply boat-maven-plugin/src/main/java/com/backbase/oss/boat/BoatBayRadio.java:59]: I guess, by default some basic security is enabled by Jhipster. If you spin up boat-bay locally, in the mysql database in jhi_user, you would see 2 users - admin(ROLE_ADMIN) and user (ROLE_USER).
To create something, we need to pass as basic auth.

Comment (bartveenstra) [reply boat-maven-plugin/src/main/java/com/backbase/oss/boat/BoatBayRadio.java:59]: No security applied

**Comment by dhananjay12** [comment]
New PR - https://github.com/Backbase/backbase-openapi-tools/pull/230

**Pull request closed without merging**